### ISSUE TITLE
Implement grunt-csscomb task

### ISSUE THREAD

**Issue by ain** [interactive-pioneers/generator-pioneerapp#44]
In conjunction of #18, `csscomb` could be implemented as a separate Grunt task in order to fill in whenever IDE is not allowing combing.

**Comment by davleh** [comment]
Yes, would be a nice thing! We could then take it over to other projects or our default grunt configuration. @beroes will also like this as he often complains about sort order and css formatting.
